### DISCUSSION TITLE
Vue installation on Winter CMS

### DISCUSSION THREAD

**Discussion by ottaviane** [wintercms/winter#258] [Q&A] [answered]
Is there any way to install vue package on Winter CMS like 'composer require laravel/ui'. I made it in a classic Laravel app. May be it usefull?
thanks. bye.

**Comment by LukeTowers** [comment, marked as answer]
Not sure what you're even asking for. What do you mean by "install vue package"? How are you expecting it to be used?

**Reply by ottaviane** [reply]
Thank you for your reply.
I hope to explain better my question:
 in a classic Laravel site with the command 'composer require laravel/ui' I install a good system who permit to  Writing Vue Components and simply with the command 'npm run dev' compile it.
This is a link that can explain better [https://levelup.gitconnected.com/how-to-set-up-and-use-vue-in-your-laravel-8-app-2dd0f174e1f8](url)

**Reply by ottaviane** [reply]
I'd like to use Vue in my pages for my site.

**Reply by bennothommo** [reply]
@ottaviane I'm not sure you would be able to use `laravel/ui` given that it requires Blade AFAIK, but you can certainly just include Vue in your theme and use standard JavaScript assets to make your frontend functionality built on top of Vue. Nothing in Winter would stop you from doing that.

**Reply by ottaviane** [reply]
Ok thanks.
